tests/unit/test_json.py

```diff
@@ -224,3 +227,76 @@ def test_json_arrow_record_batch():
         == '{"null_field":null,"order":{"address":{"city":"Anytown","street":"123 Main St"},"items":["book","pen","computer"],"total":15}}'
     )
     assert s[6] == "null"
+
+
+@pytest.fixture
+def cleanup_json_module_for_reload():
```

Comment:
I understand that this fixture makes sure db_dtypes.json is removed before any unit test that might want to import it with a clean slate, and we have added two unit tests verifying that it works. But I don't see this fixture being used in any tests. Should we add it to where we intend to use it?

Reply:
If we really use this fixture, we probably can get rid of all thee `# pragma: NO COVER` inside too.

Reply:
Removed.